integrations/common/rest/src/main/java/io/helidon/integrations/common/rest/RestApi.java

```diff
@@ -276,16 +262,15 @@ Single<T> invokeOptional(Method method,
      * @param <T> type of the built {@link io.helidon.integrations.common.rest.RestApi}
      */
     abstract class Builder<B extends Builder<B, T>, T extends RestApi> implements io.helidon.common.Builder<B, T> {
-        private final WebClient.Builder webClientBuilder = WebClient.builder()
-                .followRedirects(true)
-                .keepAlive(true);
+        private final Http1ClientBuilder webClientBuilder = Http1Client.builder();
+        //.followRedirects(true)
```

Comment:
Is there a follow up issue to fix this? Not mentioned in the PR

Reply:
https://github.com/helidon-io/helidon/issues/6804 which is already fixed

Reply:
Added redirect and keepalive.